gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Mar 07 09:42:03 CET 2017
+#Fri Jul 07 11:31:19 EEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
```

Comment:
Thanks for picking this up for review Maxime. I actually tried v4.x before pushing the branch but had problems with BB. But, it could be related to [your comment](https://github.com/wordpress-mobile/WordPress-Android/pull/6250#discussion_r126648454) so I will try again.